### ISSUE TITLE
Cannot Launch On Linux Running KDE

### ISSUE THREAD

**Issue by robkorn** [scalahub/Kiosk#1]
It appears that the functions to launch a browser use gnome bindings, and even though I installed both the libgnome2-0 and libgnome2-dev libraries on my computer running a KDE desktop, it still spits out the following error:

```
[error] (run-main-0) java.lang.UnsupportedOperationException: The BROWSE action is not supported on the current platform!
[error] java.lang.UnsupportedOperationException: The BROWSE action is not supported on the current platform!
```

If possible, can you use a more platform agnostic way of opening the browser?

**Comment by scalahub** [comment]
If you ignore this error and open http://localhost:8080 in your browser, does it work?

**Comment by robkorn** [comment]
Ah, it does indeed still work. I'm not use to sbt/scala/java so I assumed the application wouldn't work since it spat 30 lines of errors along with the 2 above.

Thanks. (Not sure if you want to close this issue, or leave it open if you intend to change the browse action so I'll leave this open for now)

**Comment by scalahub** [comment]
Yes, I'll also leave it open for now; I will see if this has a quick fix.
